scripts/data/cli.json

```diff
@@ -1920,10 +1920,10 @@
     {
       "name": "ionic doctor check",
       "namespace": ["ionic", "doctor"],
-      "summary": "\n\n:::warning\nThis command has been deprecated and will be removed in an upcoming major release of the Ionic CLI.\n:::\nCheck the health of your Ionic project",
+      "summary": "Check the health of your Ionic project",
```

Comment:
opportunity to mention the VS code extension?

Reply:
I've got a bit of work to do to understand the checks that `doctor` does and make sure that the VS Code extension covers those cases. When I get around to that I think it would be relevant to update the docs to say that it supersedes the `doctor` command.